enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/embed.ts

```diff
@@ -126,7 +131,9 @@ class MetabaseEmbed {
     }
 
     if (settings.dashboardId && settings.questionId) {
-      raiseError("can't use both dashboardId and questionId at the same time");
+      raiseError(
+        "can't use both dashboardId and questionId at the same time. to change the question to a dashboard, set the questionId to null (and vice-versa)",
+      );
```

Comment:
We should also document this in the docs that we merge new settings to existing settings for convenience, and to use `updateSettings({ questionId: null, dashboardId: 456 })` if they want to change from a question to a dashboard.